### PR TITLE
CAT-687 Merge event not updating all tables

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: database_refresh/01-configmap-refresh-script.yaml [hunk 1]
```diff
@@ -8,7 +8,7 @@ data:
     #!/bin/bash
     set -e
     # Refreshes only happen every other week, crontab doesnt really support this
-    week=$(date +%U)
+    week=$(date +%-U)
```
Comment (steverendell) [inline database_refresh/01-configmap-refresh-script.yaml:11]: unrelated typo avoiding script error

File: server/services/offendersService.js [hunk 2]
```diff
@@ -1275,11 +1275,11 @@ module.exports = function createOffendersService(nomisClientBuilder, formService
   }
 
   function inconsistentCategorisation(dbRecord, pnomisStatus) {
```
Comment (steverendell) [inline server/services/offendersService.js:1277]: unrelated slight adjustment to change treatment of 'I' records. Helpful for CAT-677 Unsentenced not appearing on recat todo list

File: server/routes/liteCategories.js [hunk 3]
```diff
@@ -46,7 +46,7 @@ const getCatList = current => [
   { value: 'T', text: 'Fem Open', selected: current === 'T' },
 ]
 
-const assessmentHasBeenApprovedInNomisMannually = error => {
+const assessmentHasBeenApprovedInNomisManually = error => {
```
Comment (steverendell) [inline server/routes/liteCategories.js:49]: typo

**Review by connormaglynn — APPROVED** [review]
lgtm

**Pull request merged**